### DISCUSSION TITLE
I want when I click the Accordion it open an accordion page not closing opening it.

### DISCUSSION THREAD

**Discussion by ervinpiol** [fuma-nama/fumadocs#617] [Q&A]
![Screenshot 2024-07-21 192241](https://github.com/user-attachments/assets/084cda87-e828-4b94-b89a-389bcaf84594)

https://github.com/user-attachments/assets/0936c657-d576-4e0e-93d8-42367e999246

**Comment by fuma-nama** [comment]
Hmm I don't quite understand your question, could you elaborate more?
